backend/lm_backend/api/routes/jobs.py

```diff
@@ -43,10 +42,13 @@ async def create_job(
     if job.cluster_client_id is None:
         job.cluster_client_id = client_id
 
-    job_created: Job = await crud_job.create(
+    job_created = await crud_job.create(
         db_session=secure_session.session, obj=JobCreateSchema(**job.dict(exclude={"bookings"}))
     )
 
+    # Appease static type checkers
```

Comment:
You removed it from elsewhere, is it still needed?

Reply:
Let me check...

Reply:
Removed in commit ed929c4